manager/manager.go

```diff
@@ -311,10 +377,31 @@ func (m *Manager) issue(ctx context.Context, volumeID string) error {
 	}
 	log.V(2).Info("Read metadata", "metadata", meta)
 
+	// check if there is already a pending request in-flight for this volume.
+	// if there is, and we still have a copy of its private key in memory, we can resume the existing request and
+	// avoid creating additional CertificateRequest objects.
+	existingReq, err := m.findPendingRequest(meta)
```

Comment:
This needs adapting to not just return a *pending* request, but also to return a *completed* request if one exists (so that it can be passed to `handleRequest` and actually issued.